bindings/stronghold-nodejs/.gitignore

```diff
@@ -0,0 +1,15 @@
+# Rust
+target/
+Cargo.lock
```

Comment:
Since we don't ship this as a Rust library a committed Cargo.lock might make sense

Reply:
We should be consistent though, so if we include the `Cargo.lock` here it should be committed with the Wasm bindings too.